specs.tex

```diff
@@ -66,6 +66,11 @@ \subsection{Arena}
         that of other zones.
   \item Tokens will be placed in the scoring zone on the opposite side to their
         matching coloured scoring zone.
+  \item \label{spec:flags} Flags will be made from wire folded in half
+        to form a single rod with a maximum width of less than \si{10}{mm},
+        and length of more than \si{500}{mm}. The length of rod without
+        a flag will be \si{400}{mm}, as described in \figref{fig:flag}.
```

Comment:
The phrase "The length of rod without a flag [is]" is not clear. I _think_ you mean "The length of rod not covered by the flag [is]", but it reads to me as "When the flag is not present, the length [is]".